tracing/optimisation/ignore.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import inspect
+
+import pandas as pd
+
+from . import Optimisation, TriggerStatus, utils
+
+
+class Ignore(Optimisation):
+    """
+    Dumb optimisation that is always ONGOING, in order to unconditionally
+    disable tracing for the relevant scope.
+    """
+
+    def __init__(self, frame: utils.FrameWithMetadata):
+        super().__init__(frame)
+        # By default, this optimisation is constantly active, as long as the current scope is active
+        self._status = TriggerStatus.ONGOING
+
+    def advance(self, current_frame: utils.FrameWithMetadata, _: pd.DataFrame) -> None:
```

Comment:
Change to update status. I would have assumed that data is passed from one advance to the next.

Reply:
`advance` refers to `updating`, just step-by-step.